content/sensu-go/5.18/guides/secrets-management.md

```diff
@@ -37,7 +37,7 @@ Secrets are configured via [secrets resources][8].
 A secret resource definition refers to the secrets provider (`Env` or `VaultProvider`) and an ID (the named secret to fetch from the secrets provider).
 
 This guide only covers the handler use case, but you can use secrets management in handler, mutator, and check execution.
-For secrets management in checks, the Sensu backend will transmit requests over its secure transport (TLS-encrypted websockets) to your Sensu agent to execute your check, so mTLS must be enabled.
+For secrets management in checks, the Sensu backend will transmit requests over its secure transport [transport layer security (TLS)-encrypted websockets] to your Sensu agent to execute your check, so you must [enable mutual TLS (mTLS)][15].
```

Comment:
```suggestion
When secrets are referenced in check configurations, the Sensu backend will only transmit execution requests for those checks to agents connected via [mutually-authenticated transport layer security (mTLS)-encrypted websockets]. Read more about [enabling mutual TLS (mTLS)][15].
```

It’s not enough for agents to be connected via TLS, their TLS session must be mutually authenticated.